src/containerapp/HISTORY.rst

```diff
@@ -3,6 +3,11 @@
 Release History
 ===============
 
+0.3.8
+++++++
+* Fixed bug with 'az containerapp update' where --yaml would error out due to secret values
```

Comment:
@haroonf this doesn't seem to have the full of list issues you fixed called out in history. Can you update please